api/middleware/validationMiddleware.js

```diff
@@ -1,13 +1,13 @@
 const validation = (schema) => async (req, res, next) => {
   const body = req.body;
   try {
-    await schema.validate(body);
-    console.log("mw.validation done");
-
+    await schema.validate(body, {
+      abortEarly: false,
+      strict: false,
+    });
     next();
-    return next()
   } catch (error) {
-    res.status(400).json({ error });
+    return res.status(400).json({ errors: error.errors });
```

Comment:
did u need this return?

Reply:
yes, I display a list of validation errors on the client

Reply:
No, thanks